lib/Transforms/IPO/PassManagerBuilder.cpp

```diff
@@ -911,6 +919,9 @@ void PassManagerBuilder::populateLTOPassManager(legacy::PassManagerBase &PM) {
     //TODO: Need to make sure the profile-use pass has been executed
     PM.add(createStructFieldCacheAnalysisPass());
   }
+
```

Comment:
Thanks! I think it's added when GitHub merging two different versions. I didn't intend to do so...